katello/hooks/post/09-mongo_syspath.rb

```diff
@@ -0,0 +1,17 @@
+def install_syspaths
+  unless facts[:operatingsystemmajrelease] == '7'
```

Comment:
Isn't this reversed logic? I think this should only be active on EL7 rather than on anything but EL7. Looks like this was already present before.

Reply:
@ekohl I tried changing it to an `if` and it didn't run:

```bash
[ WARN 2019-06-26T10:50:26 verbose]  Applied catalog in 465.94 seconds
[ INFO 2019-06-26T10:50:28 verbose] Puppet has finished, bye!
[ INFO 2019-06-26T10:50:28 verbose] Executing hooks in group post
  Success!
  * Katello is running at https://katello-nightly.katello.lan
      Initial credentials are admin / fH5Qm7wXECYdrGHE
  * To install an additional Foreman proxy on separate machine continue by running:

      foreman-proxy-certs-generate --foreman-proxy-fqdn "$FOREMAN_PROXY" --certs-tar "/root/$FOREMAN_PROXY-certs.tar"
  The full log is at /var/log/foreman-installer/katello.log
[ INFO 2019-06-26T10:50:29 verbose] All hooks in group post finished
[ INFO 2019-06-26T10:50:29 verbose] Installer finished in 481.567350482 seconds
```

Should we just remove that since it seems like the fact is not getting picked up correctly?

Reply:
Perhaps it's actually integer rather than a string? Or nil? It's there so that it shouldn't act on EL8 which has no SCLs.